src/pages/development/backward-incompatible-changes/highlights.md

```diff
@@ -11,14 +11,22 @@ This page highlights backward-incompatible changes between Adobe Commerce and Ma
 
 The following major backward-incompatible changes were introduced in the 2.4.6 Adobe Commerce and Magento Open Source releases:
 
+*  New default value for automatic redirects
 *  New system configuration for customer segments
 *  New system configuration for limiting products in grid
 *  New system configuration for OpenSearch module
 *  Symfony dependencies upgraded to latest LTS version
 *  Zend_Filter replaced with laminas-filter
 *  Zend_HTTP replaced with laminas-http
+*  Zend_Json replaced with laminas-json
 *  Zend_Validate replaced with laminas-validator
 
+### New default value for automatic redirects
+
+To improve performance, the default value for `generate_category_product_rewrites` in the [`app/code/Magento/CatalogURLRewrite/etc/config.xml`](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/CatalogUrlRewrite/etc/config.xml#L12) file was changed from `1` to `0`. This change disables [automatic redirects](https://experienceleague.adobe.com/docs/commerce-admin/marketing/seo/url-rewrites/url-redirect-product-automatic.html) by default.
```

Comment:
> This change disables automatic redirects

It's for rewrites, not redirects.  But only for rewrites with respect to the "catalog/product".

Though the weird thing is that even without the redirect,  the page still loads on my dev environment.
For example, on my local development environment, I've created subcategory "subcat2" and product "simpleproduct2" inside this category.
I can still access this new product via http://nginx.docker/index.php/subcat2/simpleproduct2.html or http://nginx.docker/subcat2/simpleproduct2.html even though I've disabled the generate_category_product_rewrites and the rewrite for this doesn't exist in `catalog_url_rewrite_product_category` and `url_rewrite`.

Reply:
Ah, yes. Thanks for correcting me on that. I'll change the link text to "rewrites" and link to the [proper section](https://experienceleague.adobe.com/docs/commerce-admin/marketing/seo/url-rewrites/url-redirect-product-automatic.html#skip-rewrite) in docs.